core-ui/src/components/Extensibility/components/Table.js

```diff
@@ -1,12 +1,33 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 
 import { GenericList } from 'shared/components/GenericList/GenericList';
 
 import { useGetTranslation } from '../helpers';
 import { Widget } from './Widget';
 
+const handleTableValue = (value, t) => {
+  switch (true) {
+    case value === undefined: {
+      return { entries: [] };
+    }
+    case Array.isArray(value): {
+      return {
+        entries: value,
+      };
+    }
+    default: {
+      return {
+        entries: [],
+        invalidConfigurationMessage: t('extensibility.widgets.table.error'),
```

Comment:
I'd go with less specific name here, so we can reuse this prop elsewhere. `genericErrorMessage`?